src/Aspire.Dashboard/Utils/CertificateUtil.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace Aspire.Dashboard;
+
+internal static class CertificateUtil
+{
+    public static X509CertificateCollection GetFileCertificate(string filePath, string? password, ILogger logger)
+    {
+        var certBytes = File.ReadAllBytes(filePath);
```

Comment:
You should be able to keep working in terms of the path, and not need to use ReadAllBytes.

`GetCertContentType` accepts path-or-bytes, but the cert load functions you need the "FromFile" variants.

The FromFile variants do things smarter than File.ReadAllBytes; but since it's a one-time/startup cost it probably doesn't matter for you.